.gitignore

```diff
@@ -13,3 +13,4 @@ test.sqlite
 *~
 *#
 coverage/
+.idea/
```

Comment:
hey - what's this `.idea/` directory?

Reply:
it's a local dev directory automagically created for projects when you're using IntelliJ IDEA. I might be the only person rolling this IDE but we don't want the dir committed.